wireapp/wire-android-reloaded
# feat: federation avs calling support (AR-1680)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1680" title="AR-1680" target="_blank"><img alt="Story" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10815?size=medium" />AR-1680</a>  Federation aware AVS calling
  </td></table>
  <br />
 






----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

We should be able to call in federated environments, with this PR, and the new kalium reference update, we are handling every case of removing. In detail, support for federated calls, passing to AVS the parsed id, this is:

- if federated, always with domain
- if not federated, without domain

Needs releases with: https://github.com/wireapp/kalium/pull/656


### Testing

#### How to Test

- Login into Anta or Bella and start a cross call, with the other user
- Calls in wire.com production should still work

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
